pep-0001.txt

```diff
@@ -610,17 +610,21 @@ optional and are described below.  All other headers are required.
 
 The Author header lists the names, and optionally the email addresses
 of all the authors/owners of the PEP.  The format of the Author header
-value must be
+values must be:
 
-    Random J. User <address@dom.ain>
+.. code-block:: text
```

Comment:
NB, I would have just added ``.. highlight:: text`` to the top of the PEP, but unlike the other languages (e.g. ``.. highlight:: rst`` in PEP 12), it doesn't change the highlight lexer of literal blocks without an explicit `text` language from `default` to `text` (which I can see comparing the DOM with and without it on this PEP vs. PEP 12), and results in this block still getting highlighted (incorrectly). Not sure if its a Sphinx (or Pygments) bug or an expected behavior I'm not understanding, but either way it seems to be due to a special case with the `text` (i.e. no highlighting) "language"/Pygments lexer.